src/Makefile.test.include

```diff
@@ -55,7 +55,9 @@ GENERATED_TEST_FILES = $(JSON_TEST_FILES:.json=.json.h) $(RAW_TEST_FILES:.raw=.r
 BITCOIN_TEST_SUITE = \
   test/main.cpp \
   test/setup_common.h \
-  test/setup_common.cpp
+  test/setup_common.cpp \
+  test/transaction_utils.h \
+  test/transaction_utils.cpp
```

Comment:
I think the file should be in a subfolder `test/lib/`, similar to `test/lib/logging` (https://github.com/bitcoin/bitcoin/pull/16540/files)

Reply:
Thanks, done. Also added the `test/lib/` folder to the Visual Studio build, like in your referenced PR.